infogami/infobase/dbstore.py

```diff
@@ -291,7 +291,7 @@ def process(c, ordering_func=None):
                     raise StopIteration
                 c.value = metadata.id
             if c.op == '~':
-                op = Literal('LIKE')
+                op = Literal('ILIKE')
```

Comment:
This will change all the places we currently use `~`, which _shouldn't_ be a problem, but maybe let's ask internally to see if anyone can foresee an error.

_Or_, if it's easy, maybe let's try to see if we can make `~i` and operator; that would let us avoid all the risk! But if it seems very difficult, maybe not worth it.